29th_first_semina/app/src/main/java/com/example/a29th_first_semina/HomeActivity.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.a29th_first_semina
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import com.example.a29th_first_semina.databinding.ActivityHomeBinding
+
+class HomeActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityHomeBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        Toast.makeText(this,"김송현님 환영합니다",Toast.LENGTH_SHORT).show()
```

Comment:
```suggestion
        Toast.makeText(this, "김송현님 환영합니다", Toast.LENGTH_SHORT).show()
```

,와 글자 사이에는 간격을 두는게 코딩 컨벤션이긴 합니다. 올리시기 전에 Ctrl+Alt+L 누르면 파일 내 텍스트 정렬 가능